src/Support/SelectFields.php

```diff
@@ -62,17 +64,24 @@ private function getFieldSelection(ResolveInfo $resolveInfo, array $args, int $d
      * Retrieve the fields (top level) and relations that
      * will be selected with the query.
      *
-     * @param  array  $queryArgs Arguments given with the query/mutation
+     * @param  array  $queryArgs  Arguments given with the query/mutation
      * @param  array  $requestedFields
      * @param  GraphqlType  $parentType
      * @param  Closure|null  $customQuery
      * @param  bool  $topLevel
+     * @param  mixed  $ctx The GraphQL context; can be anything and is only passed through
      * @return array|Closure - if first recursion, return an array,
      *               where the first key is 'select' array and second is 'with' array.
      *               On other recursions return a closure that will be used in with
      */
-    public static function getSelectableFieldsAndRelations(array $queryArgs, array $requestedFields, GraphqlType $parentType, ?Closure $customQuery = null, bool $topLevel = true)
-    {
+    public static function getSelectableFieldsAndRelations(
+        array $queryArgs,
+        array $requestedFields,
+        GraphqlType $parentType,
+        ?Closure $customQuery = null,
+        bool $topLevel = true,
+        $ctx = null
+    ) {
```

Comment:
Decided it was time to stop this "long line" madness and finally break them; did this a way times in this PR